services/transaction_body.proto

```diff
@@ -314,5 +315,10 @@ message TransactionBody {
      * Adds one or more Ed25519 keys to the affirmed signers of a scheduled transaction
      */
     ScheduleSignTransactionBody scheduleSign = 44;
+
+    /**
+     * Adds an approved allowance for spender to transfer owner's hbar.
+     */
+    CryptoApproveAllowanceTransactionBody cryptoApproveAllowance = 45;
```

Comment:
I believe you also need to add to `schedulable_transaction_body.proto`. That file looks to be missing other transactions so check with Michael if we need to add the others.